deploy/setup.sh

```diff
@@ -9,7 +9,7 @@ PIPELINE_NAME=${3:-${PIPELINE_NAME}}
 ENV_TAG=${4:-${ENV_TAG}}
 PRE_PROVISIONED_AI_MODEL_INFRA_NAME=${5:-$PRE_PROVISIONED_AI_MODEL_INFRA_NAME}
 AI_MODEL_INFRA_TYPE=${6:-${AI_MODEL_INFRA_TYPE:-"batch-account"}} # Currently supported values are aks and batch-account
-DEPLOY_PGSQL=${7:-${DEPLOY_PGSQL:-"true"}}
+DEPLOY_PGSQL=${7:-${DEPLOY_PGSQL:-"false"}}
```

Comment:
I would suggest to make this a separate change.

Reply:
Thought it's a small change so included. Any specific reason for a separate change?

Reply:
It is a change with a different intension then the original PR which is meant for batch pool job name change.

Reply:
I have reverted the flag back to its original value